docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,318 @@
+---
+# required/recommended metadata
+
+title: Datetime and DateTimeOffset Support in System.Text.Json
+description: An overview of how Datetime and DateTimeOffset types are supported in the System.Text.Json library.
+author: layomia
+ms.author: laakinri
+ms.date: 07/22/2019
+ms.topic: [TOPIC TYPE]
+ms.prod: [PRODUCT VALUE]
+helpviewer_keywords:
+  - "JSON DateTime JSON DateTimeOffset"
+  - "Serializer, Utf8"
+  - "JSON Reader, JSON Writer, times"
+
+---
+# Datetime and DateTimeOffset Support in System.Text.Json
+
+The <xref:System.Text.Json.JsonSerializer>, <xref:System.Text.Json.Utf8JsonReader>, <xref:System.Text.Json.Utf8JsonWriter>, and <xref:System.Text.Json.JsonElement> parse and write <xref:System.DateTime> and <xref:System.DateTimeOffset> text representations according to an extended profile of the ISO 8601-1:2019 format.
```

Comment:
"an extended" or "the extended"? (IIRC it's named "the extended profile" in the spec)

Reply:
More specifically **the** "basic format" does not include separators between time scale components, the "extended format" (3.1.3.5) includes separators.